pom.xml

```diff
@@ -122,8 +122,8 @@
     <commons-pool.version>1.5.7</commons-pool.version>
     <commons-digester.version>2.0</commons-digester.version>
     <commons-validator.version>1.3.1</commons-validator.version>
-    <cxf.version>2.7.0</cxf.version>
-    <jaxb-impl.version>2.2.6</jaxb-impl.version>
+    <cxf.version>2.7.17</cxf.version>
+    <jaxb-impl.version>2.3.1</jaxb-impl.version>
     <!--
```

Comment:
Not necessarily.  The plugin this comment refers to was removed since it was still having issues so I think this may be relevant if we ever try to add it back in.  I was trying to only modify things that I had to while going through this.

Reply:
Ah, ok. You might consider updating it to make that clear because it reads to me like it's referring to the property below.

Reply:
Are you talking about updating the WorkflowDocumentService? I'm just saying this comment no longer makes sense as written since it refers to us downgrading the plugin when we're now on an even newer version. I still think it would be helpful to have an update in the comment that explains what you mentioned above: that the problem causing plugin has been removed and that the comment remains for historical purposes.

Reply:
Turns out that property was unused so I removed it.